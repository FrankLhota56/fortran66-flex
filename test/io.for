C
C Read / Write tests
C
C23456
      Read (5, 1010) A, (B(i), I=1,3)
      Write (6, 1020) (B(I), I=1,5,2), c
      Read (5)
      Write (6) name
      end
