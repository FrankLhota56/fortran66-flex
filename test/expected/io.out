Line     5: READ
Line     5: '('
Line     5: INT_LIT(5)
Line     5: ','
Line     5: INT_LIT(1010)
Line     5: ')'
Line     5: ID(a)
Line     5: ','
Line     5: '('
Line     5: ID(b)
Line     5: '('
Line     5: ID(i)
Line     5: ')'
Line     5: ','
Line     5: ID(i)
Line     5: '='
Line     5: INT_LIT(1)
Line     5: ','
Line     5: INT_LIT(3)
Line     5: ')'
Line     6: EOS
Line     6: WRITE
Line     6: '('
Line     6: INT_LIT(6)
Line     6: ','
Line     6: INT_LIT(1020)
Line     6: ')'
Line     6: '('
Line     6: ID(b)
Line     6: '('
Line     6: ID(i)
Line     6: ')'
Line     6: ','
Line     6: ID(i)
Line     6: '='
Line     6: INT_LIT(1)
Line     6: ','
Line     6: INT_LIT(5)
Line     6: ','
Line     6: INT_LIT(2)
Line     6: ')'
Line     6: ','
Line     6: ID(c)
Line     7: EOS
Line     7: READ
Line     7: '('
Line     7: INT_LIT(5)
Line     7: ')'
Line     8: EOS
Line     8: WRITE
Line     8: '('
Line     8: INT_LIT(6)
Line     8: ')'
Line     8: ID(name)
Line     9: EOS
Line     9: END
